docs/workflow/requirements/linux-requirements.md

```diff
@@ -40,6 +40,14 @@ Install the following packages for the toolchain:
 * zlib1g-dev
 * ninja-build (optional, enables building native code with ninja instead of make)
 
+You can install all the above dependencies by running
+
+```bash
+sudo xargs apt-get install -Y < eng/debian-reqs.txt
```

Comment:
Worth adding a similar line here for other package managers? I know I would copy paste it.

Reply:
We now just have a shell script to run.